### DISCUSSION TITLE
Welcome to unstoppable-wallet-ios Discussions!

### DISCUSSION THREAD

**Discussion by horsys** [horizontalsystems/unstoppable-wallet-ios#4437] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Blessme102** [comment]
Please does unstoppable app support dogecoin? Thanks

**Reply by Dianaaiym** [reply]
> Please does unstoppable app support dogecoin? Thanks

Hi, DOGE isn’t supported yet

**Comment by heshangok** [comment]
In the source code I do not know how to open the NFT function，can you help me

**Reply by Mycatty** [reply]
Ok it's too
